examples/04_glm_first_level/plot_two_runs_model.py

```diff
@@ -0,0 +1,271 @@
+"""
+Simple example of two-runs fMRI model fitting
+================================================
+
+Here, we will go through a full step-by-step example of fitting a GLM
+to experimental data and visualizing the results.
+This is done on two runs of one subject of the FIAC dataset.
+
+This is a block design experiment with a 2 X 2 experimental design
+with the following factors:
+
+- ``Sentence`` with 2 levels: Same (SSt) vs Different (DSt)
+- ``Speaker``  with 2 levels: Same (SSp) vs Different (DSp)
+
+giving the 4 following conditions:
+
+- Same Sentence-Same Speaker (SStSSp)
+- Same Sentence-Different Speakers (SStDSp)
+- Different Sentences-Same Speaker (DStSSp)
+- Different Sentences-Different Speakers (DStDSp)
+
+The design also included a 5th condition
+containing the first sentence pooled across all conditions.
```

Comment:
Added this section but this could / should probably go in the descrption of the dataset (that is currently missing see #4210)

Reply:
Indeed. For a later PR.